terminus-ui/icon/src/custom-icons/csv.ts

```diff
@@ -2,8 +2,33 @@
+        <path d="M10,0 L2,0 C0.9,0 0.01,0.9 0.01,2 L0,18 C0,19.1 0.89,20 1.99,20 L14,20 C15.1,20 16,19.1 16,18 L16,6 L10,0 Z M2,18 L2,2 L9,2
```

Comment:
FYI All of these internal SVG changes are just me formatting them to respect our max-line-length